controls/radspellchecker/getting-started.md

```diff
@@ -84,8 +88,26 @@ And the one for DataGrid is telerik:DataGridSpellCheckHelper.IsSpellCheckingEnab
 	      </sdk:DataGridTemplateColumn>
 	   </sdk:DataGrid.Columns>
 	</sdk:DataGrid>
-	{{endregion}}
-
+{{endregion}}
 
 
 Of course the GridViewDataColumn (or DataGridTemplateColumn) should contain a single control that can be spellchecked (implements IControlSpellChecker interface). [ReadMore]({%slug radspellchecker-spellchecking-gridcell%})
+
+
+## Using RadSpellChecker with Implicit Styles
+
+When the theme is [implictly set]({%slug styling-apperance-implicit-styles-overview%}) in your applciation you need to merge the required dictionaries. RadSpellChecker has a dependency on the __Telerik.Windows.Controls.Navigation.dll__, which means that you should merge the __Telerik.Windows.Documents.Proofing.xaml__ as well as the __Telerik.Windows.Controls.Navigation.xaml__ file:

```

Comment:
RadSpellChecker has a dependency on the __Telerik.Windows.Controls.Navigation.dll__, which means that you should merge the __Telerik.Windows.Documents.Proofing.xaml__ as well as the __Telerik.Windows.Controls.Navigation.xaml__ file - this sentence sounds strange after the update of the code. Please, think of rewriting it.